src/components/Header/Header.jsx

```diff
@@ -202,6 +202,7 @@ export function Header(props) {
     sessionStorage.removeItem('viewingUser');
     window.dispatchEvent(new Event('storage'));
     props.getWeeklySummaries(user.userid)
+    window.location.href = `/dashboard/${userId}`;
```

Comment:
**Suggestion for code improvement**: Using window.location.href reloads the entire application, which can be inefficient. The useHistory hook from React Router is better suited for this case as it enables navigation without a full reload. 

logged-in user ID should not visible in the URL and Url should be '/dashboard' which maintains consistency throughout the application.

Reply:
Resolving the conversation as this is fixed in changeset 201ad4f1abf550ff2669fe79c1cb7466e7afe291